portfolio-alex.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+
+<html>
+	
+	<head>
+		
+		<link rel="stylesheet" type="text/css" href="portfolio-alex.css"/>
+		<link href="https://fonts.googleapis.com/css?family=Spartan&display=swap" rel="stylesheet"/>
+		
+		<meta name="viewport" content="width=device-width,initial-scales=1"/>
+		
+		<title>Alex Jennings Portfolio</title>
+		
+	</head>	
+	
+	<body>
+			
+		<header>
+				
+			<section class="topheader">
+					
+				<div class="logo">
+						
+					<div class="logotext">
+					
+						<h1>ALEX JENNINGS</h1>
+						
+						<p>Full Stack Developer</p>
+							
+					</div>
+				
+				</div>
+				
+		 	 	 	<nav class="navlinks">
+					
+						<div class="navholder">
+	  
+	  						<strong><a href="#aboutme">ABOUT ME</a>
+	  		 						<a href="#portfolio">PORTFOLIO</a>
+	  		 						<a href="#contacts">CONTACTS</a></strong>
```

Comment:
This is still a bit weird formatting wise, something like this would be more readable:
```html
<strong>
   <a>
</strong>
```